components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -702,12 +702,14 @@ define([
                 var result = this._generateEmptyField(data);
                 var cancerList = editor.getCancerLegend()._getAllSupportedCancers();
 
-                var div = new Element('div', {'class': 'cancer_field cancer-header field-no-user-select'} );
-                var label1 = new Element('label', {'class': 'cancer_label_field'} ).update("Name");
-                var label2 = new Element('label', {'class': 'cancer_status_select'} ).update("Status");
-                var label3 = new Element('label', {'class': 'cancer_age_select'} ).update("As of");
+                var div = new Element('tr', {'class': 'cancer_field cancer-header field-no-user-select'} );
```

Comment:
even better would be to give them meaningful names. Like headerContainer or something